spec/models/comment_spec.rb

```diff
@@ -11,22 +11,39 @@
       @comment = Comment.new(body: 'My comment', post: @post, user_id: 10000)
     end
 
-    it "sends an email to users who have favorited the post" do
-      @user.favorites.where(post: @post).create
+    context "with user's permission" do
 
-      allow(FavoriteMailer)
-        .to receive(:new_comment)
-        .with(@user, @post, @comment)
-        .and_return(double(deliver:true))
+      it "sends an email to users who have favorited the post" do
+        @user.favorites.where(post: @post).create
 
-      @comment.save
+        allow(FavoriteMailer)
+          .to receive(:new_comment)
+          .with(@user, @post, @comment)
+          .and_return(double(deliver:true))
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Space missing after colon.